src/Common/Core/Cookie.php

```diff
@@ -139,26 +139,6 @@ private function normalizeDomain(string $domain = null): ?string
         return null;
     }
 
-    /**
-     * Has the visitor allowed cookies?
-     *
-     * @return bool
-     */
-    public function hasAllowedCookies(): bool
-    {
-        return $this->get('cookie_bar_agree', 'N') === 'Y';
```

Comment:
I would really love to put this in Fork 5 but this is a BC break. 
Can you implement it in a way that in Fork 5 you can choose for the cookie bar or the consent dialog? And then just add a deprecation notice on the cookie bar

Reply:
Fixed in c96374567c5503b16ae091b4121883be50ce9d41